src/js/electron/brim.ts

```diff
@@ -69,7 +70,9 @@ export class BrimMain {
   }
 
   activate() {
-    if (this.windows.count() === 0) this.windows.init()
+    log.info("activate fired! window count: ", this.windows.count())
+    if (this.windows.count() === 0 || this.windows.count() === 1)
+      this.windows.init()
```

Comment:
Let's add a method called `this.windows.getVisible().length` I think that would help this read more clearly.